### DISCUSSION TITLE
❓ [Question] Is it ok to build v1.1.0 with cuda10.2 not default cuda11.3?

### DISCUSSION THREAD

**Discussion by wikiwen** [pytorch/TensorRT#1255] [Q&A] [answered]
## ❓ Question

Is it ok to build v1.1.0 with cuda10.2 not default cuda11.3?
It's hard to upgrade latest gpu driver for some machine which is shared by many people. So cuda10.2 is preferred.

## Environment

> Build information about Torch-TensorRT can be found by turning on debug messages

 - PyTorch Version: 1.11.0
 - CUDA version: 10.2 
 - TensorRT:  8.2.4.2

**Comment by narendasan** [comment, marked as answer]
You can build Torch-TensorRT with 10.2 as long as you change the dependencies for CUDA, cuDNN and TensorRT, LibTorch to pull 10.2 based builds. 

- https://github.com/pytorch/TensorRT/blob/db61e904f5e92219a6d3ce06f343670185ec886c/WORKSPACE#L44
- https://github.com/pytorch/TensorRT/blob/db61e904f5e92219a6d3ce06f343670185ec886c/WORKSPACE#L61
- https://github.com/pytorch/TensorRT/blob/db61e904f5e92219a6d3ce06f343670185ec886c/WORKSPACE#L69
- https://github.com/pytorch/TensorRT/blob/db61e904f5e92219a6d3ce06f343670185ec886c/WORKSPACE#L82
- https://github.com/pytorch/TensorRT/blob/db61e904f5e92219a6d3ce06f343670185ec886c/WORKSPACE#L92
